packages/server/modules/core/graph/resolvers/branches.js

```diff
@@ -1,142 +1,18 @@
-const getBranchById = getBranchByIdFactory({ db })
-const getStreamBranchByName = getStreamBranchByNameFactory({ db })
-const createBranchAndNotify = createBranchAndNotifyFactory({
-  getStreamBranchByName,
-  createBranch: createBranchFactory({ db }),
-  addBranchCreatedActivity
-})
-const updateBranchAndNotify = updateBranchAndNotifyFactory({
-  getBranchById,
-  updateBranch: updateBranchFactory({ db }),
-  addBranchUpdatedActivity
-})
-const deleteBranchAndNotify = deleteBranchAndNotifyFactory({
-  getStream,
-  getBranchById: getBranchByIdFactory({ db }),
-  modelsEventsEmitter: ModelsEmitter.emit,
-  markBranchStreamUpdated,
-  addBranchDeletedActivity,
-  deleteBranchById: deleteBranchByIdFactory({ db })
-})
-const getUser = legacyGetUserFactory({ db })
-
 /** @type {import('@/modules/core/graph/generated/graphql').Resolvers} */
 module.exports = {
-  Query: {},
-  Stream: {
-    async branches(parent, args) {
-      return await getPaginatedStreamBranches(parent.id, args)
-    },
-
-    async branch(parent, args) {
-      // TODO: TEMPORARY HACK
-      // Temporary "Forwards" compatibility layer to allow .NET and PY clients
-      // to use FE2 urls without major changes.
-      // When getting a branch by name, if not found, we try to do a 'hail mary' attempt
-      // and get it by id as well (this would be coming from a FE2 url).
-
-      const branchByName = await getStreamBranchByName(parent.id, args.name)
-      if (branchByName) return branchByName
-
-      const branchByIdRes = await getBranchById(args.name)
-      if (!branchByIdRes) return null
-
-      // Extra validation to check if it actually belongs to the stream
-      if (branchByIdRes.streamId !== parent.id) return null
-      return branchByIdRes
-    }
-  },
-  Branch: {
-    async author(parent, args, context) {
-      if (parent.authorId && context.auth) return await getUser(parent.authorId)
-      else return null
-    }
-  },
-  Mutation: {
-    async branchCreate(parent, args, context) {
-      await authorizeResolver(
-        context.userId,
-        args.branch.streamId,
-        Roles.Stream.Contributor,
-        context.resourceAccessRules
-      )
-
-      const { id } = await createBranchAndNotify(args.branch, context.userId)
-
-      return id
-    },
-
-    async branchUpdate(parent, args, context) {
-      await authorizeResolver(
-        context.userId,
-        args.branch.streamId,
-        Roles.Stream.Contributor,
-        context.resourceAccessRules
-      )
-
-      const newBranch = await updateBranchAndNotify(args.branch, context.userId)
-      return !!newBranch
-    },
-
-    async branchDelete(parent, args, context) {
-      await authorizeResolver(
-        context.userId,
-        args.branch.streamId,
-        Roles.Stream.Contributor,
-        context.resourceAccessRules
-      )
-
-      const deleted = await deleteBranchAndNotify(args.branch, context.userId)
-      return deleted
-    }
-  },
   Subscription: {
```

Comment:
Subscription resolvers can't be moved to TS 1:1, they require a different syntax and new type definitions which can only be resolved by using the debugger and looking at real runtime values. Will clean these up later

Reply:
I like that done in a dedicated PR
BTW, if you find this is too much work, we can skip it.
Moving to TS is outside of our current scope